event_hub.gemspec

```diff
@@ -32,7 +32,9 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   # Uncomment to register a new dependency of your gem
-  # spec.add_dependency "example-gem", "~> 1.0"
+  spec.add_dependency "bunny", '>= 2.13.0'
+  spec.add_dependency "aws-sdk-sns", '~> 1.0'
+  spec.add_dependency "aws-sdk-sqs", '~> 1.0'
```

Comment:
looks like it makes sense to create 2 more gems: `event_hub_aws` and `event_hub_bunny` so if the project uses only AWS it will not install bunny gem and vice versa